packages/cli/lib/utils.ts

```diff
@@ -141,42 +152,48 @@ export async function upgradeAction(debug = false) {
             printDebug(`Latest version of ${resolved.name} is ${latestVersion}.`);
         }
 
-        if (semver.gt(latestVersion, version)) {
-            console.log(chalk.red(`A new version of ${resolved.name} is available: ${latestVersion}`));
-            const cwd = process.cwd();
+        if (!semver.gt(latestVersion, version)) {
+            state.set('lastIgnoreUpgrade', Date.now());
```

Comment:
In case the last check was negative we also skip, it will fasten the cli calls in general

Reply:
Can you remove this piece of logic -- otherwise looks good

Reply:
fixed